### PR TITLE
Add basic README.md

### PR DESCRIPTION
**Pull request by premun**
I tried to cover the basics that shouldn't much anymore, but for sure we owe more details on requirements once things settle

Fixes: #12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+To list all the possible commands, use the `help` command:
+
+```console
+dotnet xharness help
+```
+
+To get help for a sub-command command:
+
+```console
+dotnet xharness ios test help
```
Comment (premun) [inline README.md:50]:
```suggestion
dotnet xharness ios test --help
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+# XHarness
+
+This repo contains the code to build the **XHarness dotnet tool**.
+
+## What is XHarness
+
+XHarness is primarily a command line tool that enables running tests on Android and iOS (WatchOS and tvOS are also supported).
+It can locate devices/emulators, install given application, run it and collect results (uninstall it after).
```
Comment (MattGal) [inline README.md:8]: grammar nits: "install a given application", "uninstalling it after" or "and uninstall it after"

File: README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+It handles crashes gracefully and supports different types of connection modes (Network, USB cable).
```
Comment (MattGal) [inline README.md:9]: Gracefully is a subjective term and doesn't mean anything.  If you're going to say how it handles crashes it should be more explicit, like "Handles bad machine state or crashing APKs by detecting this, logging as much as possible ... yada"

File: README.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+It can output test result in various different formats from text to Xunit/NUnit XML.
```
Comment (MattGal) [inline README.md:10]: It can output test results

File: README.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+The tool can also package given .NET test DLLs (Xunit, NUnit v2/3) into an iOS app bundle that can be ran on the device/emulator.
```
Comment (MattGal) [inline README.md:12]: nit: can be run

File: README.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+## Running the tool
+
+The tool requires **.NET 3.1.201** and later to be run. It is packaged as a `dotnet tool` command and can be installed using the [dotnet tool CLI](https://docs.microsoft.com/en-us/dotnet/core/tools/).
+
+- The iOS scenarios require you to run the tool on MacOS with Xcode 11.4.
+- Android part is supported on both Linux and Windows devices
```
Comment (MattGal) [inline README.md:19]: why parts and scenarios?  pick a word and stick with it.

File: README.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+To install the tool run:
+
+```console
+dotnet tool install --global --add-source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json Microsoft.DotNet.XHarness.CLI --version 1.0.0-prerelease.20229.6
```
Comment (MattGal) [inline README.md:24]: can we leave off version so they get latest?  Not sure how that works but the doc will be instantly old.

Comment (MattGal) [reply README.md:24]: nm you mentioned this.

**Comment by MattGal** [comment]
> Please address the language comments from @MattGal but as a foreign speaker, it looks good.

It's all just suggestions, plenty of non-ESL folks have terrible grammar here and as long as it doesn't obfuscate the meaning of what's being said it's purely me enjoying copy editing.

**Pull request merged**